requirements.txt

```diff
@@ -1,8 +1,7 @@
-psutil>=5.6.7
-pytest>=5.3.1
-pandas>=0.25.3
-tqdm>=4.38.0
-pyyaml>=5.1.2
-bs4>=4.8.2
-lxml>=4.4.2
-
+beautifulsoup4>=4.12.2
+numpy>=1.21.6
+pandas>=1.4.3
+python_dateutil>=2.8.2
+requests>=2.28.1
+setuptools>=65.6.3
+tqdm>=4.65.0
```

Comment:
Thanks for the updating the versions here! I installed it on python 3.10 and it seems to work perfectly.